android/apollo/src/com/andrew/apollo/cache/ImageCache.java

```diff
@@ -286,8 +286,7 @@ private Bitmap getBitmapFromMemCache(final String data) {
             return null;
         }
         if (mLruCache != null) {
-            final Bitmap lruBitmap = mLruCache.get(data);
```

Comment:
I'll merge these, they might seem stupid, but the reason this code is here is for debugging purposes, this way you can pause the debugger within this function and know what the output was.

Reply:
Ah, I see.